src/utils/parser/utils.js

```diff
@@ -99,8 +99,12 @@ function getFunctionName(path) {
     return parent.id.name;
   }
 
-  if (parent.right && isFunction(parent.right)) {
-    return parent.left.property.name;
+  if (parent.type == "AssignmentExpression") {
+    if (parent.left.type == "MemberExpression") {
+      return parent.left.property.name;
+    }
+
+    return parent.left.name;
```

Comment:
I tweaked this block to:

* remove `isFunction(parent.right)` which i believe is unnecessary
* prioritize the `MemberExpression` which is consistent with the other type checks first